assets/less/cds-rdm/globals/site.overrides

```diff
@@ -88,15 +88,93 @@
     background-color: @recordsListAltBackground;
   }
 
-  .metadata-background {
-    background-color: @recordsListMetadataBackground;
-  }
-
   .description{
     font-size: 1rem;
   }
 }
 
+#communities-carousel{
+
+  img {
+    @media all and (min-width: @largestMobileScreen) {
+      position: absolute;
+      left: -1em;
+    }
+    top: -2em;
+    border: 1px solid @black;
+  }
+
+  .ui.container.carousel{
+    background-color: @white;
+
+    & > .full-width {
+      width: 100% !important;
+      max-width: none;
+
+      & > .max-width-computer {
+        max-width: @computerBreakpoint !important;
+
+        &.center{
+          margin-left: auto;
+          margin-right: auto;
+
+          @media all and (min-width: @computerBreakpoint) {
+            text-align: right;
+          }
+          @media all and (max-width: @computerBreakpoint) {
+            text-align: center;
+          }
+        }
+      }
+    }
+  }
+
+  .ui.items {
+
+    & > .item {
+
+      position:relative;
+      border-style: inset !important;
```

Comment:
this style causes a 3D effect around the item, which was not present on the mockup. please use the default border style without inset.